src/components/ModularTable/ModularTable.test.tsx

```diff
@@ -42,6 +42,18 @@ it("renders all rows and columns", async () => {
   });
 });
 
+it("renders all rows and columns when the sortable option is set", async () => {
```

Comment:
Thanks, sorry, I haven't noticed your previous comment about giving you some tips.

Would it be possible to replicate a simple test like this one from `MainTable`?
https://github.com/canonical/react-components/blob/main/src/components/MainTable/MainTable.test.tsx#L177-L187

At least to verify "aria-sortable" is added to column headers. We can treat that as a minimal confirmation of the `sortable` feature doing something.

If you don't have time, I can do it myself sometime later this week to finalize and merge your PR.

Reply:
Made a test following the basic scheme of that linked test case. Thanks!